master/static/srv/proto_get_trial_ids_for_experiment.sql

```diff
@@ -0,0 +1,66 @@
+WITH searcher_info AS (
+    SELECT config->'searcher'->>'metric' AS metric_name,
+           (
+               SELECT CASE
+                          WHEN coalesce(
+                                  (
+                                              config->'searcher'->>'smaller_is_better'
+                                      )::boolean,
+                                  true
+                              ) THEN 1
+                          ELSE -1
+                          END
+           ) AS sign
+    FROM experiments e
+    WHERE e.id = $1
+), filtered_experiment_trials AS (
+    SELECT
+        t.id AS id,
+        'STATE_' || t.state AS state,
+        t.start_time,
+        t.end_time,
+        coalesce(t.end_time, now()) - t.start_time AS duration,
+        (
+            SELECT s.prior_batches_processed + s.num_batches
+            FROM steps s
+            WHERE s.trial_id = t.id
+                AND s.state = 'COMPLETED'
+            ORDER BY s.id DESC
+            LIMIT 1
+        ) AS total_batches_processed,
```

Comment:
:) wondering if we should also make this a separate column.

Reply:
heh. yeah, sgtm!